### DISCUSSION TITLE
Lodestar Standup 2024-6 (February 13, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6418] [Standups]
### Meeting Info

[Feb 13, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/feb-13-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
Led by @wemeetagain 
- Planning Items
  - v1.16.0 Planning
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by jeluard** [comment]
Discuss lodestar strategy regarding external dependencies (including implicit dependencies on `nodejs`).
2 points come to mind:

* reduce dependencies on 3rd parties (security, audit)
* help with `browser` support

It might also make sense to discuss dependencies between lodestart packages (e.g `utils` has a number of dependencies that will be transitively included).

**Reply by nazarhussain** [reply]
For `lightclient` and `prover` packages it's very important to have lightweight browser compatible builds.

**Comment by nazarhussain** [comment]
We may decide some action point on the skipped tests, which are over 20 at the moment. https://github.com/ChainSafe/lodestar/discussions/6376#discussioncomment-8333098

**Reply by matthewkeil** [reply]
Perhaps we do a PR that removes the skipped tests (and no other changes) and then everyone on the team can look at the ones that are being removed in one go.  And we can all push to the single PR branch if tests need to be turned back on, updated or give the blessing to remove.

I still like the idea of moving skips to a "skip" file so they are in one place and more importantly we can run one if its prudent for a specific use-case on a PR but not a useful test outside of the limited set of reasons.

**Comment by philknows** [comment]
Closed in favour of #6434
